pkg/taskrunmetrics/metrics.go

```diff
@@ -455,12 +480,24 @@ func (r *Recorder) RunningTaskRuns(ctx context.Context, lister listers.TaskRunLi
 	}
 	metrics.Record(ctx, runningTRsCount.M(float64(runningTrs)))
 	metrics.Record(ctx, runningTRs.M(float64(runningTrs)))
-	metrics.Record(ctx, runningTRsThrottledByNodeCount.M(float64(trsThrottledByNode)))
-	metrics.Record(ctx, runningTRsThrottledByQuotaCount.M(float64(trsThrottledByQuota)))
 	metrics.Record(ctx, runningTRsWaitingOnTaskResolutionCount.M(float64(trsWaitResolvingTaskRef)))
-	metrics.Record(ctx, runningTRsThrottledByNode.M(float64(trsThrottledByNode)))
-	metrics.Record(ctx, runningTRsThrottledByQuota.M(float64(trsThrottledByQuota)))
+	metrics.Record(ctx, runningTRsThrottledByQuotaCount.M(float64(trsThrottledByQuotaCount)))
+	metrics.Record(ctx, runningTRsThrottledByNodeCount.M(float64(trsThrottledByNodeCount)))
 
+	for ns, cnt := range trsThrottledByQuota {
+		ctx, err = tag.New(ctx, []tag.Mutator{tag.Insert(namespaceTag, ns)}...)
```

Comment:
This doesn't behave properly for `metrics.taskrun.throttle.enable-namespace` : false.

Reply:
good catch ... I've updated the code path and unit test to cover both flavors of the new config option - thanks